internal/campaign/campaign.go

```diff
@@ -170,3 +190,75 @@ func readPrDescriptionFile(filename string) (string, string, error) {
 
 	return prTitle, strings.Join(prBodyLines, "\n"), nil
 }
+
+func CreateInitialFiles(campaignName string) error {
+	data := TemplateVariables{
+		CampaignName: campaignName,
+	}
+	files := map[string]string{
+		".gitignore": gitignoreTemplate,
+		".turbolift": turboliftTemplate,
+		"README.md":  readmeTemplate,
+		"repos.txt":  reposTemplate,
+	}
+	for filename, templateFile := range files {
+		err := ApplyTemplate(filepath.Join(campaignName, filename), templateFile, data)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// Applies a given template and data to produce a file with the outputFilename
+func ApplyTemplate(outputFilename string, templateContent string, data interface{}) error {
```

Comment:
Do all these functions need to be exported outside the package (capital letter at beginning of name)?

Reply:
good point, that one doesn't. The others are all called from different packages.